src/main/java/io/github/swagger2markup/internal/component/PathOperationComponent.java

```diff
@@ -328,8 +330,65 @@ private void exampleMap(MarkupDocBuilder markupDocBuilder, Map<String, Object> e
                 buildExampleTitle(markupDocBuilder, sectionTitle + " " + entry.getKey());
-                markupDocBuilder.listingBlock(Json.pretty(entry.getValue()), "json");
+
+                if (NumberUtils.isNumber(entry.getKey())) {
+                    // Section header is an HTTP status code (numeric)
+                    try {
+                        JsonFactory factory = new JsonFactory();
+                        ObjectMapper mapper = new ObjectMapper(factory);
+                        JsonNode rootNode = mapper.readTree(Json.pretty(entry.getValue()));
+
+                        Iterator<Map.Entry<String, JsonNode>> fieldsIterator = rootNode.fields();
+
+                        if (!fieldsIterator.hasNext()) {
+                            // workaround for "array" example
+                            //TODO: print $ref'd examples correctly instead of just "array"
+                            String example = Json.pretty(entry.getValue())
+                                    .replaceAll("^\"+", "")  // Strip leading quotes
+                                    .replaceAll("\"+$", ""); // Strip trailing quotes
+                            example = Json.pretty(example);
+                            markupDocBuilder.listingBlock(example, "json");
+                        }
+                        while (fieldsIterator.hasNext()) {
+                            Map.Entry<String, JsonNode> field = fieldsIterator.next();
+
+                            if (field.getKey().equals("application/json")) {
+
+                                String example = Json.pretty(field.getValue());
+                                example = StringEscapeUtils.unescapeJson(example)
+                                        .replaceAll("^\"+", "")  // Strip leading quotes
+                                        .replaceAll("\"+$", ""); // Strip trailing quotes
+
+                                markupDocBuilder.listingBlock(example, "json");
+
+                            } else if (field.getKey().equals("application/xml")) {
+                                String example = field.getValue().toString()
+                                        .replaceAll("^\"+", "")  // Strip leading quotes
+                                        .replaceAll("\"+$", ""); // Strip trailing quotes
+
+                                example = StringEscapeUtils.unescapeJava(example);
+
+                                //TODO: pretty print XML
+
+                                markupDocBuilder.listingBlock(example, "xml");
+                            } else {
+                                String example = Json.pretty(entry.getValue());
+                                markupDocBuilder.listingBlock(example, "json");
+                                break; // No need to print the same example multiple times
+                            }
+                        }
+                    } catch (Exception ex) {
+                        //TODO: Actually handle exception
```

Comment:
Upon further inspection the only exception I believe we could get here is an IOException. Perhaps we could propagate it up so the user can get the complete exception.